cddl/crypto-key-type-choice.cddl

```diff
@@ -1,9 +1,13 @@
 $crypto-key-type-choice /= tagged-pkix-base64-key-type
 $crypto-key-type-choice /= tagged-pkix-base64-cert-type
 $crypto-key-type-choice /= tagged-pkix-base64-cert-path-type
+$crypto-key-type-choice /= tagged-cose-key-type
 $crypto-key-type-choice /= tagged-thumbprint-type
+$crypto-key-type-choice /= tagged-cert-thumbprint-type
 
 tagged-pkix-base64-key-type = #6.554(tstr)
 tagged-pkix-base64-cert-type = #6.555(tstr)
 tagged-pkix-base64-cert-path-type = #6.556(tstr)
 tagged-thumbprint-type = #6.557(digest)
+tagged-cose-key-type = #6.558(COSE_KeySet / COSE_Key)
```

Comment:
```suggestion
tagged-cose-key-type = #6.558(COSE_Key)
```
I don't see yet why COSE_KeySet should be supported. Everywhere `$crypto-key-type-choice` is used, it already is contained in an array. Also every `$crypto-key-type-choice` definition contains data for a single key or cert, it would be good to keep the new definition `tagged-cose-key-type` consistent.

Reply:
Since RFC8152 defines both `COSE_Key` and `COSE_KeySet`, after further consideration, I think it makes sense to support both, to align with the RFC.

Reply:
Since RFC8152 defines the structure, the COSE_Key CDDL doesn't need to be incorporated into the prose. However, the cddl file is useful for building examples.